airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/source.py

```diff
@@ -132,10 +132,9 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
         Show error message in case of request exception or unexpected response.
         """
         try:
-            config = AmazonSellerPartnerConfig.parse_obj(config)  # FIXME: this will be not need after we fix CDK
             stream_kwargs = self._get_stream_kwargs(config)
             orders_stream = VendorSalesReports(**stream_kwargs)
-            next(orders_stream.read_records(sync_mode=SyncMode.full_refresh))
+            next(orders_stream.read_records(None))
```

Comment:
Which argument do we set to `None`? And why should we consider changing the value of `sync_mode`?

Reply:
The first arg is the `sync_mode`, it's mandatory, if `None` is passed to it the `full-refresh` is applied by default.

Reply:
So why we need change this?

Reply:
Using `sync_mode=SyncMode.full_refresh` is clearer and more explicit than setting it to silence None.

Reply:
to simplify the unnecessary imports, no problem, I'll revert it back.

Reply:
Anyway it is better to use a named argument when set to None or comment to be more clearer.